src/main/java/org/folio/qm/controller/RecordsEditorRecordsApi.java

```diff
@@ -1,14 +1,15 @@
 package org.folio.qm.controller;
 
-import java.util.UUID;
-
+import org.folio.qm.domain.dto.QuickMarc;
+import org.folio.qm.rest.resource.RecordsApi;
+import org.folio.qm.service.MarcRecordsService;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import org.folio.qm.domain.dto.QuickMarc;
-import org.folio.qm.rest.resource.RecordsApi;
-import org.folio.qm.service.MarcRecordsService;
+import javax.validation.Valid;
+import java.util.UUID;
```

Comment:
Imports are in incorrect order in all classes. The order should be:
- import static *
- import static org.folio.*
- import java/javax.*
- import  *
- import org.folio.*